### ISSUE TITLE
[GUI] A synchronized file might appear as unsynchronized in the GUI

### ISSUE THREAD

**Issue by vxgmichel** [Scille/parsec-cloud#312]
Steps to reproduce:

 - run the test environment and the GUI
   ```shell
   $ source misc/run_test_environment.sh; parsec core gui -l DEBUG
   ```
 - log in as `alice@pc`
 - open the `alice_workspace` workspace in the gui
 - create a new file using the file manager or the shell
   ```shell
   $ touch ~/parsec_mnt/alice-alice_workspace/test1  # Then test2, test3, etc.
 - repeat until the freshly created file appears in the GUI as unsynchronized
 - note that the file has been synchronized by looking at the logs:
   ```
   2019-03-15 14:16:28 [debug] Send event [parsec.event_bus] event_name=fs.entry.synced id=UUID('89a95a58-7ec3-4b78-8486-075adc966064') path=/alice_workspace/test3
   ```
 - Note that leaving and entering the workspace again from the GUI updates the file status to synchronized.

**Comment by Max-7** [comment]
Probably linked to #262

**Comment by Max-7** [comment]
Seems fixed by #389. I created 20 files following your indications and all appear as synced.
